streaming/java/streaming-api/src/main/java/org/ray/streaming/api/context/StreamingContext.java

```diff
@@ -48,8 +48,12 @@ public void execute() {
     this.plan = planBuilder.buildPlan();
     plan.printPlan();
 
-    IJobSchedule jobSchedule = new JobScheduleImpl(jobConfig);
-    jobSchedule.schedule(plan);
+    ServiceLoader<IJobSchedule> serviceLoader = ServiceLoader.load(IJobSchedule.class);
+    Iterator<IJobSchedule> iterator = serviceLoader.iterator();
+    Preconditions.checkArgument(iterator.hasNext());
```

Comment:
This code will throw an exception with no info.

source code from guava:
  public static void checkArgument(boolean expression) {
    if (!expression) {
      throw new IllegalArgumentException();
    }
  }

Reply:
fixed